sql-plugin/src/main/scala/com/nvidia/spark/rapids/Plugin.scala

```diff
@@ -146,6 +147,20 @@ object RapidsPluginUtils extends Logging {
     }
     // set driver timezone
     conf.set(RapidsConf.DRIVER_TIMEZONE.key, ZoneId.systemDefault().normalized().toString)
+
+    // If spark.rapids.sql.multiThreadedRead.numThreads is not set explicitly, or it is set
+    // explicitly as value 0, then we derive it from other settings. Otherwise, we keep the
+    // users' setting.
+    val numThreadsKey = RapidsConf.MULTITHREAD_READ_NUM_THREADS.key
+    if (!conf.contains(numThreadsKey) || conf.get(numThreadsKey).toInt == 0) {
+      // Derive it from spark.executor.cores
+      // In standalone mode, users set spark.executor.cores via `--conf spark.executor.cores=10`.
+      // In yarn mode, users set spark.executor.cores via a config file 'spark-default.conf',
```

Comment:
this is actually not completely true, you can set spark.executor.cores via command line as well.  I think both of these comments are not needed so please remove.   If we were to say anything here, I would say that spark.executor.cores is not set on all cluster managers by default.

Reply:
I have updated the comments.